native/fastlane/Fastfile

```diff
@@ -159,13 +159,13 @@ end
 platform :ios do
   private_lane :apple_auth do |options|
     ensure_env_vars(
-        env_vars: ["APP_STORE_CONNECT_API_KEY_ID", "APP_STORE_CONNECT_API_ISSUER_ID", "APP_STORE_CONNECT_API_KEY"]
+        env_vars: ["APP_STORE_CONNECT_API_KEY_ID", "APP_STORE_CONNECT_API_ISSUER_ID", "APP_STORE_CONNECT_API_KEY_CONTENT"]
```

Comment:
With fastlane version v2.175.0 the environment variable APP_STORE_CONNECT_API_KEY is used internally. Setting it manually results in conflicting builds and errors.
https://github.com/fastlane/fastlane/issues/18324